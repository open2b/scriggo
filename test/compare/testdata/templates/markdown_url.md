{# render #}

http://www.open2b.com/
https://www.open2b.com/
http://{{ "www.open2b.com" }}/
https://{{ "www.open2b.com" }}/

https://www.open2b.com?a=b
https://www.open2b.com?a={{ "b" }}
https://www.open2b.com?a={{ "=" }}

 https://www.open2b.com?a={{ "=" }}
-https://www.open2b.com?a={{ "=" }}
"https://www.open2b.com?a={{ "=" }}"
\https://www.open2b.com?a={{ "=" }}

https://www.open2b.com?a={{ "=" }}.{{ "?" }}
https://www.open2b.com?a={{ "=" }}. ?
