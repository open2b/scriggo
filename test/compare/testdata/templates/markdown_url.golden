
http://www.open2b.com/
https://www.open2b.com/
http://www.open2b.com/
https://www.open2b.com/

https://www.open2b.com?a=b
https://www.open2b.com?a=b
https://www.open2b.com?a=%3d

 https://www.open2b.com?a=%3d
-https://www.open2b.com?a=%3d
"https://www.open2b.com?a=%3d"
\https://www.open2b.com?a=%3d

https://www.open2b.com?a=%3d.%3f
https://www.open2b.com?a=%3d. ?
